test-requirements.txt

```diff
@@ -19,4 +19,5 @@ tensorflow
 torch
 torchvision
 pysftp
-keras
+# TODO: don't pin Keras version once https://github.com/keras-team/keras/issues/11276 is addressed
+keras==2.2.2
```

Comment:
Should we force users running version < `2.2.2` to upgrade? We can also do `keras<=2.2.2`.

Reply:
I think only travis should need to install these requirements. Keras isn't a dependency when you do `pip install mlflow`. Those are found in `setup.py`!

Reply:
Whoops - got it!